store/datastore.go

```diff
@@ -50,6 +53,15 @@ type DataStore interface {
 	DeleteDevice(ctx context.Context, deviceID string) error
 	SetIntegrationCredentials(context.Context, uuid.UUID, model.Credentials) error
 	RemoveIntegration(context.Context, uuid.UUID) error
+
+	// GetAllDevices returns an iterator over ALL devices sorted by tenant ID.
```

Comment:
I do not entirely follow sorting by tenant ID, it is just a dictionary asc sort?

Reply:
I've utilized the fact that the devices are returned sorted (order doesn't really matter) in the caching for the CLI command.